package.json

```
@@ -82,13 +82,15 @@
     "**/hoist-non-react-statics": "^3.3.2",
     "**/isomorphic-fetch/node-fetch": "^2.6.1",
     "**/istanbul-instrumenter-loader/schema-utils": "1.0.0",
+    "**/json-schema": "^0.4.0",
     "**/minimist": "^1.2.5",
     "**/node-jose/node-forge": "^0.10.0",
     "**/pdfkit/crypto-js": "4.0.0",
     "**/react-syntax-highlighter": "^15.3.1",
     "**/react-syntax-highlighter/**/highlight.js": "^10.4.1",
     "**/trim": "1.0.1",
     "**/typescript": "4.1.3",
+    "**/uglify-js": "^3.14.3",
```

I believe you can change this to be more specific so it won't affect the uglify 2.x dependency:

```suggestion
    "**/handlebars/uglify-js": "^3.14.3",
```

---

Wouldn't customers still get the security warning for the outdated uglify-js version then though?

---

To clarify, the uglify-js@^3.1.4 and uglify-js@3.4.x package versions are dev-only dependencies, which we aren't concerned about upgrading.

The uglify-js@^3.14.3 and uglify-js@^2.6.2 package versions are production dependencies. It looks like uglify-js@^3.14.3 is upgradable and uglify-js@^2.6.2 is not. So we should just upgrade uglify-js@^3.14.3 to the latest.

---

I've not been able to bump these without custom resolutions but versions should be correct now.